client_generic/Networking/Networking.cpp

```diff
@@ -288,8 +290,11 @@ bool	CCurlTransfer::Perform( const std::string &_url )
     if( !Verify( curl_easy_setopt( m_pCurl, CURLOPT_SSL_VERIFYPEER, 0 ) ) )	return false;
 
 	Status( "Active" );
-	//if( !Verify( curl_easy_perform( m_pCurl ) ) )
+#if defined(_MSC_VER) && (_MSC_VER == 1900)		
+	if( !Verify( curl_easy_perform( m_pCurl ) ) )

```

Comment:
Why we need to fall back to curl_easy_perform ? It is non-interruptible, so the screensaver can get stuck on download when somebody need to close it in the middle of download. The whole InterruptiblePerform is there for a reason. Can you please explain why you need to use curl_easy_perform ?